browser/app/js/buckets/actions.js

```diff
@@ -33,7 +33,6 @@ export const fetchBuckets = () => {
   return function(dispatch) {
     return web.ListBuckets().then(res => {
       const buckets = res.buckets ? res.buckets.map(bucket => bucket.name) : []
-      dispatch(setList(buckets))
```

Comment:
this shouldn't be removed instead moved into `if (buckets.length > 0)`  - check the reference fix in the previous comment